doc/API/centreon-api-v2.1.yaml

```diff
@@ -2689,6 +2689,86 @@ paths:
           $ref: '#/components/responses/Forbidden'
         '500':
           $ref: '#/components/responses/InternalServerError'
+  /platform/topology/{poller_id}:
+    delete:
+      tags:
+        - Topology
+      parameters:
+        - $ref: '#/components/parameters/PollerId'
+      summary: 'Delete a Platform'
+      description: "Delete a platform from the topology"
+      responses:
+        '204':
+          description: "Platform deleted"
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+  /gorgone/pollers/{poller_id}/responses/{token}:
```

Comment:
Are you sure, that gorgone endpoints are supported on v2.1 ?
that these routes actually exists in v2.1 conf ?

Reply:
Yes